blog/_posts/2017-09-15-Team-Alexa-update.md

```diff
@@ -0,0 +1,74 @@
+---
+title: The end of Sasha&Sasha's journey
```

Comment:
Hi, thanks for your PR. By mistake you've added one extra file : .Rhistory. Can you please remove it?